python/sklearnserver/sklearnserver/model.py

```diff
@@ -0,0 +1,33 @@
+import kfserving
+from joblib import load
+import numpy as np
+import os
+
+
+class SKLearnModel(kfserving.KFModel):
+    def __init__(self, name, model_dir):
+        self.name = name
+        self.model_dir = model_dir
+        self.ready = False
+
+    def load(self):
+        model_file = kfserving.Storage.download(self.model_dir)
+        self._model = load(model_file)
```

Comment:
can you import as something other than load to avoid name conflict confusion

Reply:
Done